aviator.yml

```diff
@@ -0,0 +1,17 @@
+spruce:
+  - base: ci/meta.yml
+    prune:
+      - meta
+    merge:
+      - with_in: ci/
+        regexp: ".*yml"
+        except:
+          - credentials.yml.example
+          - credentials.yml
```

Comment:
remove these 3 lines, I'm guessing they been copied around for a couple of years now